3rdparty/celt-0.7.0-build/CMakeLists.txt

```diff
@@ -68,3 +68,11 @@ target_sources(celt PRIVATE
 )
 
 target_disable_warnings(celt)
+
+find_library(MATH_LIBRARY m)
+
+# If -lm exists
+if(MATH_LIBRARY)
+	# Link it to prevent "contains an unresolvable reference to symbol sin: it's probably a plugin"
+	target_link_libraries(celt PUBLIC ${MATH_LIBRARY})
+endif()
```

Comment:
```suggestion

# We have to explicitly link to the "m" (math) library.
# See https://stackoverflow.com/q/1033898/3907364 for why
find_library(MATH_LIBRARY m)

# If -lm exists
if(MATH_LIBRARY)
	# Link it to prevent "contains an unresolvable reference to symbol sin: it's probably a plugin"
	target_link_libraries(celt PUBLIC ${MATH_LIBRARY})
endif()
```